src/main/resources/assets/securitycraft/models/block/reinforced_block.json

```diff
@@ -1,4 +1,5 @@
 {
+    "gui_light": "side",
```

Comment:
It was in the minecraft/model/block/block.json file, so I though it would be no harm if it's also in the reinforced_block.json file. But I can still remove it if you want to

Reply:
No, it's ok.